client/js/mousehandling.js

```diff
@@ -122,7 +122,12 @@ async function inputHandler(name, e) {
       mouseStatus[target.id].coords = coords;
       if(mouseStatus[target.id].moveTarget) {
         setDeltaCause(`${playerName} dragged ${widget.id}`);
-        await mouseStatus[target.id].moveTarget.move(coords, mouseStatus[target.id].localAnchor);
+        const mt = mouseStatus[target.id].moveTarget;
+        if(mt.get('type') === 'zoomy' && mt.isZoomed()) {
+          await mt.moveZoomed(coords, mouseStatus[target.id].localAnchor);
+        } else {
+          await mt.move(coords, mouseStatus[target.id].localAnchor);
+        }
```

Comment:
This hack isn't necessary. You can define a new `move` for the zoomy class that checks if it's zoomed and if not, calls `super.move(...)` instead.

Reply:
Fixed.